### ISSUE TITLE
Test user migration on dev and push to live

### ISSUE THREAD

**Issue by chrismclarke** [OneArmyWorld/onearmy#166]
A system has been put in place to help migrate legacy users to the new site. The dev database has been populated with legacy user data which should enable people who had old accounts to log into the new site.

This needs to be tested, and if working then the additional database migrated onto live to support the same functionality, and periodically populated with any new users from the legacy site.

@mattia, can you try get a few people to log into the dev site and see if it allows them using their old username and password (the process may take around 5s or so)

requires #185

**Comment by mattia-io** [comment]
Did some tests but wasn't working. Here is the error. Let me know when I can try again.
![Screen Shot 2019-01-08 at 12.40.24.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5beadf2e66e72100136491d9/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5700b77226454fb6b7191a379ed0c2f51415d4abe5d4a1f06a3a3dc78e2a436628f0e5abe823c5672ce6f43a9b9811fbf07eb258a4a86b4bd9850d2aa6c7bb3d4d216744b470a83ed5c32aa984ecc5ff44ae64f57a412336af43b.png)

**Comment by chrismclarke** [comment]
requires #185 - I will try to find time for this next weekend and get back to you when ready again

**Comment by chrismclarke** [comment]
Given previous discussions and some recent investigation a slightly better system should probably be put in place, consisting of:

1. Create a new account (makes it easier if everyone does this)
2. Migrate user profile data from Dave Hakkens on click (make overt)

Will refactor this a little today and just waiting on Dave to upload a plugin on the wordpress site to make accessing user profile image possible

**Comment by chrismclarke** [comment]
system redesigned and basic implementation and testing complete. Could still do with some tidying and refining but will cover in separate issue later and close here for now
